capnpy/testing/compiler/test_distutils.py

```diff
@@ -89,7 +91,8 @@ def test_setup_build(self, monkeypatch, ROOT):
         ret = os.system('%s setup.py build_ext --inplace' % sys.executable)
         assert ret == 0
         if self.pyx:
-            outfile = self.tmpdir.join('example.so')
+            outfile = self.tmpdir.join(
+                'example' + (sysconfig.get_config_var('EXT_SUFFIX') or '.so'))
```

Comment:
since we use this logic twice, maybe we can put it at top-level and save the result in a constant